apps/api/src/form/form.service.ts

```diff
@@ -1,25 +1,55 @@
-import { Injectable } from '@nestjs/common';
+import { Inject, Injectable, Logger, LoggerService } from '@nestjs/common';
 import { Repository } from 'typeorm';
 import { FormEntity } from './entity/form.entity';
 import { InjectRepository } from '@nestjs/typeorm';
-import { FormDTO } from '@ehpr/common';
+import { FormDTO, PersonalInformationDTO } from '@ehpr/common';
 import { booleanToYesNo } from 'src/common/helper/csv/casting';
 import { FormExportColumns } from 'src/common/helper/csv/formExport';
+import { MailService } from 'src/mail/mail.service';
+import { ConfirmationMailable } from 'src/mail/mailables/confirmation.mailable';
+import { Recipient } from 'src/mail/types/recipient';
 
 @Injectable()
 export class FormService {
   constructor(
+    @Inject(Logger) private readonly logger: LoggerService,
     @InjectRepository(FormEntity)
     private readonly formRepository: Repository<FormEntity>,
+    @Inject(MailService)
+    private readonly mailService: MailService,
   ) {}
   async saveForm(dto: FormDTO, confirmationId: string): Promise<FormEntity> {
     const newForm = this.formRepository.create({
       ...dto,
       confirmationId,
     } as Partial<FormEntity>);
 
-    return await this.formRepository.save(newForm);
+    const savedForm = await this.formRepository.save(newForm);
+
+    const notifiedForm = await this.sendMail(savedForm);
+
+    return notifiedForm;
+  }
+
+  private async sendMail(form: FormEntity) {
```

Comment:
`sendMail` seems too generic and sounds like it should be in the mail module, does this function need to be moved or should it be `sendFormConfirmation[Email]`?

Reply:
Good idea